modules/puppet_proxy/puppet_config.rb

```diff
@@ -1,4 +1,4 @@
-require 'augeas'
+require 'augeas' if RUBY_PLATFORM =~ /ruby/
```

Comment:
Hmm, I'm quite certain this isn't the case. Make sure puppet, puppetca, and bmc modules are disabled in their respective config files.